edge-modules/azure-monitor/src/IotHubUpload/IotHubUpload.cs

```diff
@@ -29,10 +29,19 @@ public async Task<bool> PublishAsync(IEnumerable<Metric> metrics, CancellationTo
             try
             {
                 Preconditions.CheckNotNull(metrics, nameof(metrics));
-
                 IEnumerable<ExportMetric> outputMetrics = metrics.Select(m => new ExportMetric(m));
 
-                byte[] metricsData = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(outputMetrics));
+                string outputString = JsonConvert.SerializeObject(outputMetrics);
+                LoggerUtil.Writer.LogDebug("Metrics selected for upload . . .");
+                LoggerUtil.Writer.LogDebug(outputString);
```

Comment:
These log statements, combined with the next set of statements below, could produce a lot of output. Is that what we want?

Reply:
I thought those are debug logs and will work only with debug deployment manifest. I can remove it if it's not the case

Reply:
OK, I figured that was the case, just double-checking. 👍

Reply:
Commented out the LogDebug lines for now. For debugging, we can always uncomment the lines.